### PR TITLE
Ports/psoc6 docs general info

### PR DESCRIPTION
**Pull request by jaenrig-ifx**
I want to add a picture to the release section.
And a better guidance on the script part, which needs to be completed when we have something working in that way.
If we plan to release before that, then I will remove those sections.
But as a first draft for feedback:
https://ifx-micropython.readthedocs.io/en/ports-psoc6_docs-general-info/psoc6/tutorial/intro.html

### CODE REVIEW

**Review by NikhitaR-IFX — APPROVED** [review]
Fairly easy to understand for me right now.

**Review by IFX-Anusha — APPROVED** [review]
Good enough for getting started with psoc & mpy

File: docs/psoc6/tutorial/intro.rst [hunk 1]
```diff
@@ -1,4 +1,131 @@
...
+
+In order to deploy the firmware in the embedded device, the easiest and quickest way is to use the
+`Cypress Programmer <https://softwaretools.infineon.com/tools/com.ifx.tb.tool.cypressprogrammer>`_.
+It allows to flash a binary in the PSoC6™ microcontrollers family in a few clicks from your Windows,
+Linux or Mac OS machine. 
+Follow the instructions on the provided link to download and install the tool.
+
+After that, select the downloaded MicroPython firmware *.hex* file to be deployed on the PSoC6™. Then, in
+the upper menu, select the connected *Probe/Kit*, click on *Connect*, and finally click on *Program*.
+The log section will show the progress and inform when the firmware deployment osn the controller is completed.
```
Comment (IFX-Anusha) [inline docs/psoc6/tutorial/intro.rst:51]: typo -  firmware deployment on the controller

File: docs/psoc6/tutorial/intro.rst [hunk 2]
```diff
@@ -1,4 +1,131 @@
...
+Follow the instructions on the provided link to download and install the tool.
+
+After that, select the downloaded MicroPython firmware *.hex* file to be deployed on the PSoC6™. Then, in
+the upper menu, select the connected *Probe/Kit*, click on *Connect*, and finally click on *Program*.
+The log section will show the progress and inform when the firmware deployment on the controller is completed.
+
+.. image:: ../img/cy-programmer.jpg
+    :alt: Cypress Programmer GUI
+    :width: 360px
+
```
Comment (boramonideep) [inline docs/psoc6/tutorial/intro.rst:56]: Just a suggestion: my brain would like the image ceter aligned :) 

```
:align: center
```

File: docs/psoc6/tutorial/intro.rst [hunk 3]
```diff
@@ -1,4 +1,131 @@
...
+Use your preferred serial terminal software to connect to the board. Examples of serial
+terminal tools are `Putty <https://www.putty.org/>`_, which works for Windows and
+Unix machines; or other platform specific such as `Tera Term <https://ttssh2.osdn.jp/index.html.en>`_, or `minicom <https://en.wikipedia.org/wiki/Minicom>`_. 
+
+Configure the serial connection with **115200 bauds** and **8-N-1** (8 bits frame, no parity and 1 stop
+bit), and connect to the board port, the MicroPython REPL prompt will appear, and you can start
+typing some python code :) 
+
+.. image:: ../img/mpy-psoc6-repl.jpg
```
Comment (boramonideep) [inline docs/psoc6/tutorial/intro.rst:85]: also
```
:align: center
```

**Review by boramonideep — APPROVED** [review]

**Review by bargfred — APPROVED** [review]

File: docs/psoc6/tutorial/intro.rst (hunk 2 above)
Comment (jaenrig-ifx) [reply docs/psoc6/tutorial/intro.rst:56]: Might be... but just to keep the same style, as no other pic is aligned center in the docs, let´s keep it like that.
Maybe making a bit bigger helps?

Comment (boramonideep) [reply docs/psoc6/tutorial/intro.rst:56]: Yes, making it a little bit bigger would be better.

**Pull request merged**